packages/lesswrong/lib/collections/reviewWinners/cache.ts

```diff
@@ -34,6 +35,6 @@ export async function getPostReviewWinnerInfo(postId: string, context: ResolverC
 
   return (
     REVIEW_WINNER_CACHE.reviewWinnersByPostId[postId] ??
-    await context.ReviewWinners.findOne({ postId })
+    await context.ReviewWinners.findOne({ postId, createdAt: {$lt: moment(`${BEST_OF_LESSWRONG_PUBLISH_YEAR}-01-01`).toDate()} })
```

Comment:
I don't think we're supposed to be filtering on `createdAt` here, rather than `reviewYear: { $lte: BEST_OF_LESSWRONG_PUBLISH_YEAR }` (unless you're donig something different from what I think you're doing)

Reply:
Yeah I figured that out and drafted the PR until I fixed it.